pkg/utils/utils.go

```diff
@@ -109,6 +118,16 @@ func BuildCSPDirectives(k8sMode, pluginsCSP, indexPageScriptNonce string) ([]str
 	scriptSrcDirective = append(scriptSrcDirective, []string{unsafeEval, nonce}...)
 	styleSrcDirective = append(styleSrcDirective, unsafeInline)
 
+	// If the objectSrc directive is not set, add 'none' to it.
+	if len(objectSrcDirective) == 1 && objectSrcDirective[0] == "object-src" {
+		objectSrcDirective = append(objectSrcDirective, none)
+	}
+
+	// If the connectSrc directive is not set, add 'none' to it.
+	if len(connectSrcDirective) == 1 && connectSrcDirective[0] == "connect-src" {
+		connectSrcDirective = append(connectSrcDirective, none)
+	}
```

Comment:
I think specifying `none` overrides `default-src` so things like `self` would no longer work.

Reply:
ok, I see, in that case we should only set `self`, for both object-src` and `connect-src`